recipes/vsgimgui/all/patches/0.1.0-0001-includeMacro.patch

```diff
@@ -0,0 +1,98 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index eb0a39c..1ae1ff5 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -23,34 +23,11 @@ if (VULKAN_SDK)
+ endif()
+ 
+ find_package(vsg 1.0.5)
++include(${VSG_MACRO_PATH}/vsgMacros.cmake)
```

Comment:
This should not be necessary if we declare `cmake_build_modules` property in vsg for that macro

Reply:
I'll update the vsg package and then re-integrate this